working-together/README.md

```diff
@@ -41,6 +41,19 @@ and valued during remote meetings that happen over video conferencing software.
   - Consider calling in if you have an unreliable internet connection.
   - Be understanding of interruptions that may happen with remote work,
     e.g. children, pets, other people who might be sharing space with your teammate.
+- **Cultural Diversity:** In a diverse workplace, people will have diverse cultures, accents and forms of speech, and this is totally fine. Always try to make communication as clear as possible.
+  - **Avoid** whenever possible:
+    - Using slang when talking about an important subject
+    - Using passive voice (i.e. *"You will be asked some questions"*)
+    - Ambiguous phrases or words with multiple meanings
+    - Harsh sentences, even if they don't sound rude in certain cultures (i.e. *"Why can't you do it? It's easy."*)
+  - **Prioritize** whenever possible:
+    - Short, direct sentences over long sentences
```

Comment:
I think we can link this article https://pudding.cool/2022/02/plain/ somewhere.

Also, feel free to pull more examples from it!

Reply:
Thank you! Gonna check it out!